docs/02-providers/aws/README.md

```diff
@@ -41,8 +41,20 @@ provider:
             - AWS::EC2::Instance
 ```
 
-### Deployment S3Bucket
-The bucket must exist beforehand and be in the same region as the deploy.
+### Deployment S3 Bucket
+The bucket must exist beforehand and be in the same region as the deploy. Due to the importance and hard dependency of the deployment bucket, if you want to switch between the core and custom bucket, you have to do the following:
```

Comment:
"The bucket must exist beforehand and be in the same region as the Lambda functions you want to deploy."